packages/utils/src/modern.js

```diff
@@ -15,22 +12,29 @@ export const ModernBrowsers = {
   'Mobile Safari': '10.3'
 }
 
-const modernBrowsers = Object.keys(ModernBrowsers)
-  .reduce((allBrowsers, browser) => {
-    allBrowsers[browser] = semver.coerce(ModernBrowsers[browser])
-    return allBrowsers
-  }, {})
+const modernBrowsers = new Proxy(ModernBrowsers, {
```

Comment:
Still, we are always initializing a Proxy. What if we instead use `getModernBrowsers` which in the first call evaluates the value and keep in `_modernBrowsers` for next calls?

Reply:
Proxy get won't be called at initialization, it's same as using _modernBrowsers

Reply:
Would also prefer using memoization here. Seems cleaner then using a Proxy

Reply:
Makes sense :)

Reply:
I also like Proxy self-memo pattern :)

Reply:
The only thing that should check later is that ensure rollup doesn't count this as a side-effect and tree-shaking works for the const variable.

Reply:
Actually my first edition is using a local state `__modernBrowsers` and function `getModernBrowsers`, but I feel having a `__modernBrowsers` in scope   
 is still kind of side effect, and proxy can have shorter code and fine-grained control.

Reply:
The difference is in `__modernBrowsers` method, side-effect is only an empty object rather than a proxy that also has getter logic itself and less low-level. But bundle size is not really a concern so i'm ok with trying new Proxy approach :)

Reply:
I checked the built dist/utils.js, it works fine.
@pimlie Maybe you can check with your usage ?

Reply:
Mergeed with [release] to beta test :)

Reply:
@clarkdo Doesnt seem to work:
```
 ERROR  Webpack build for lambda failed [                                                                                 11:56:45
  '../node_modules/@nuxt/utils-edge/dist/utils.js\n' +
    "Module not found: Error: Can't resolve 'semver/functions/coerce' in 'nuxt-lambda/node_modules/@nuxt/utils-edge/dist'",
  '../node_modules/@nuxt/utils-edge/dist/utils.js\n' +
    "Module not found: Error: Can't resolve 'semver/functions/gte' in 'nuxt-lambda/node_modules/@nuxt/utils-edge/dist'"
]
```

Reply:
@clarkdo This is because I had still stubbed semver btw, let me check what happens if i dont stub it

Reply:
When I dont stub semver I get a runtime error, have to check it. But nuxt/utils load is as follows:

![image](https://user-images.githubusercontent.com/1067403/74101735-520ea080-4b3d-11ea-8077-a82e30de51e9.png)

Total time is 8.2ms for loading/parsing @nuxt/utils-edge. The webpack require is signal-exit, the small yellow block is `escapeRegExp` and the other block already shows the function.

Reply:
From log, the error is from `isModernBrowser`, `semver` is required at hat time for browser detection.

Reply:
@pimlie I've applied the changes from your and @pi0 review, but it's not released to edge